tools/tdbg/util.go

```diff
@@ -63,10 +63,10 @@ func prettyPrintJSONObject(o interface{}) {
 	fmt.Println()
 }
 
-func getRequiredGlobalOption(c *cli.Context, optionName string) (string, error) {
+func getRequiredOption(c *cli.Context, optionName string) (string, error) {
 	value := c.String(optionName)
```

Comment:
Looks like in cli v2 library, there's no `GlobalString` method and `String` can be used to get both local and global flags. So only one version of get required option is needed for both local and global flags.